logging/runtime/src/main/java/io/quarkiverse/googlecloudservices/logging/runtime/LoggingConfiguration.java

```diff
@@ -0,0 +1,220 @@
+package io.quarkiverse.googlecloudservices.logging.runtime;
+
+import java.util.Map;
+import java.util.Optional;
+
+import com.google.cloud.logging.Severity;
+import com.google.cloud.logging.Synchronicity;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "google.cloud.logging", phase = ConfigPhase.RUN_TIME)
+public class LoggingConfiguration {
+
+    /**
+     * Which Google Operations log should be used by default.
+     */
+    @ConfigItem
+    public String defaultLog;
+
+    /**
+     * Enable or disable the Goocle Cloud logging.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
+
+    /**
+     * Configure base formatting to be either plain text or
+     * structured json.
+     */
+    @ConfigItem(defaultValue = "TEXT")
+    public LogFormat format;
+
+    /**
+     * Configure GCP logging synchronicity.
+     */
+    @ConfigItem
+    public Optional<Synchronicity> synchronicity;
+
+    /**
+     * Configure auto flush level.
+     */
+    @ConfigItem
+    public Optional<ConfigLevel> flushLevel;
+
+    /**
+     * Configure default labels.
+     */
+    @ConfigItem
+    public Map<String, String> defaultLabel;
+
+    /**
+     * Configured the monitored resource. Please consult the Google
+     * documentation for the correct values: https://cloud.google.com/logging/docs/api/v2/resource-list#resource-types
```

Comment:
Use `@see` javadoc snippet for clickable links

Reply:
Check.